lib/php/admin-bundle/Form/JsonType.php

```diff
@@ -24,14 +26,31 @@ public function transform(mixed $value)
 
     public function reverseTransform(mixed $value)
     {
+        if ($value !== null && json_validate($value) === false) {
+            return ['input-error' => 'Invalid JSON: ' . json_last_error_msg()];
```

Comment:
I believe `{"input-error": "Invalid JSON: ..."}` will appear in the field?
The user must see its first (errored) input to correct it.

Reply:
No, it's in the reverseTransform, data will be insert to DB, the user input not change